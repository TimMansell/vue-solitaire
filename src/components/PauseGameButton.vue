<template>
  <Button
    :is-stacked="true"
    :check-disabled="true"
    route="pause"
    data-test="pause-game-btn"
  >
    Pause
  </Button>
</template>

<script>
import Button from './Button.vue';

export default {
  name: 'PauseGameButton',
  components: {
    Button,
  },
};
</script>
