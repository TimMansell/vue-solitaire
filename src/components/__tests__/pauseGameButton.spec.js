import { describe, it, expect } from 'vitest';
import { shallowMount } from '@vue/test-utils';
import PauseGameButton from '@/components/PauseGameButton.vue';

describe('PauseGameButton.vue', () => {
  it('renders the component without crashing', () => {
    const wrapper = shallowMount(PauseGameButton);

    expect(wrapper.isVisible()).toBe(true);
  });
});
