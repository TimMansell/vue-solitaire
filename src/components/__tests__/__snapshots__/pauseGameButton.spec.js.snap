// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PauseGameButton.vue matches snapshot 1`] = `
<button-stub type="default" isstacked="true" checkdisabled="true" size="md" topage="pause" data-test="pause-game-btn">
  Pause
</button-stub>
`;
